reactor-core/src/main/java/reactor/core/publisher/SignalLogger.java

```diff
@@ -201,6 +201,13 @@ else if (level == Level.SEVERE) {
 	 * @see #log
 	 */
 	void safeLog(SignalType signalType, Object signalValue) {
+		if (signalValue instanceof Fuseable.QueueSubscription) {
+			signalValue = String.valueOf(signalValue);
```

Comment:
Useless outside of "if" (if debug is not enabled)

Reply:
what do you mean? The idea is to always convert the `QueueSubscription` into a `String` before passing it to the concrete logging library, and _if DEBUG is enabled_ log an additional context message.

Reply:
Ooops, sorry. Was reading on a phone, didn't spotted it!